src/components/displaybooks.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable */
+import React from 'react';
+import style from './displaybooks.module.css';
+
```

Comment:
- [ ] The path to the CSS module is `'/displaybook.module.css` and not `'./displaybooks.module.css'`. Please pay attention to the `s` at the end of displaybook and remove it. 👍💻